HTML/mockup-minimo/index.html

```diff
@@ -0,0 +1,185 @@
+                </picture>
+                <h3 class="article__tag">photodiary</h3>
+                <h2 class="article__header">A day exploring the Alps
+                </h2>
+                <p class="article__paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
+                    et
+                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                </p>
+            </article>
+        </section>
+        <section class="section-sign-in">
+            <h2 class="section-sign-in__header article__header">Sign up for our newsletter!</h2>
+            <form class="section-sign-in__form">
+                <input type="email" class="section-sign-in__input" placeholder="Enter a valid email address">
+                <input type="submit" class="section-sign-in__button" value=&rarr;>
+            </form>
+        </section>
+        <section class="section-posts">
+            <div class="section-posts--second">
+                <article>
+                    <picture>
+                        <source media="(min-width: 1200px)" srcset="img/image_5@3x-min.png">
+                        <source media="(min-width: 500px)" srcset="img/image_5@2x-min.png">
+                        <img src="img/image_5-min.png" class="article__image" alt="article photo">
+                    </picture>
+                    <h3 class="article__tag">lifestyle</h3>
+                    <h2 class="article__header">Top 10 song for running
+                    </h2>
+                    <p class="article__paragraph">
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et
+                        dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                    </p>
+                </article>
+                <article>
+                    <picture>
+                        <source media="(min-width: 1200px)" srcset="img/image_6@3x-min.png">
+                        <source media="(min-width: 500px)" srcset="img/image_6@2x-min.png">
+                        <img src="img/image_6-min.png" class="article__image" alt="article photo">
+                    </picture>
+                    <h3 class="article__tag">lifestyle</h3>
+                    <h2 class="article__header">Cold winter days
+                    </h2>
+                    <p class="article__paragraph">
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et
+                        dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                    </p>
+                </article>
+            </div>
+            <div class="section-posts__button">
```

Comment:
I wouldn't use button inside of posts as generally it is button to load new posts but not part of them